core/deployment/pom.xml

```diff
@@ -18,6 +18,10 @@
             <groupId>org.aesh</groupId>
             <artifactId>readline</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
```

Comment:
Is this needed for this branch ?

Reply:
Yes, this module depends on it directly but currently it is apparently pulled in as a transitive dependency from `org.apache.maven` dependencies of the bootstrap. Which means if a Gradle project excludes all the `org.apache.maven` deps, this specific artifact will missing from the classpath.

Reply:
ok got it.

Reply:
A bit late to the party but do we really want a dependency to commons-lang in our core artifacts?

Reply:
IIRC, a utility class is used in dev services.